File: AsusAPI/Model/Domain/Kupac.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Model.Domain
{
    [Table("Kupac")]
    public class Kupac
    {
        [Key]
        public int PIB { get; set; }
        public string NazivKupca { get; set; }
        public string UlicaBroj { get; set; }
        public int IDDrzave { get; set; }
        public int PostanskiBroj { get; set; }
        public Grad Grad { get; set; }


    }
}

File: ASUS-Frontend/src/app/_model/kupac.ts
import { Drzava } from "./drzava";
import { Grad } from "./grad";

export interface Kupac {
    PIB:number,
    NazivKupca:string,
    UlicaBroj:string,
    IDDrzave:number,
    PostanskiBroj:number,
    Grad:Grad,
    Drzava: Drzava
}
